tensornetwork/backends/tensorflow/tensorflow_backend_test.py

```diff
@@ -629,3 +629,15 @@ def test_power(dtype):
 def test_eps(dtype):
   backend = tensorflow_backend.TensorFlowBackend()
   assert backend.eps(dtype) == tf.experimental.numpy.finfo(dtype).eps
+
+def test_pinv():
+  backend = tensorflow_backend.TensorFlowBackend()
+  np.random.seed(10)
+  tensor = np.random.rand(2, 3, 4)
+  rcond = np.random.rand(1)[0] * 1E-14
+  expected = tf.linalg.pinv(tensor, rcond)
+  actual = backend.pinv(tensor, rcond)
+  np.testing.assert_allclose(expected, actual)
+  expected = tf.linalg.pinv(tensor)
+  actual = backend.pinv(tensor)
+  np.testing.assert_allclose(expected, actual)
```

Comment:
add a test that `actual @ tensor` and `tensor @ actual` equal the identity (within the desired error)

Reply:
Would testing for all the criteria in https://en.wikipedia.org/wiki/Moore%E2%80%93Penrose_inverse be a better test? (As for an example given in tf's documentation, the product wasn't identity)